astropy/astropy.cfg

```diff
@@ -60,6 +60,13 @@
 ## How many times to retry communications when they fail
 # n_retries = 10
 
+[constants]
+
+## Default version of physical constants
+# physical_constants = 'codata2014'
+
+## Default version of astronomical constants
+# astronomical_constants = 'iau2015'
```

Comment:
I think we shouldn't actually use config items here but ScienceState, as described here:

http://docs.astropy.org/en/stable/config/index.html#adding-new-configuration-items

Reply:
I'm afraid I am not seeing how ScienceState can lead to consistent initialization of units, which is importing constants. It seems to me that config items for constants are needed before the first import of units.

Reply:
Unless...as part of ScienceState, we do an `import lib.reload` of every core package that imports from constants?

Reply:
In writing a test for the configuration items: I found  I can modify the `astropy.constants` configuration, then reload the `constants` package, and I get the desired version of constants. Unfortunately, I tried `importlib.reload(astropy.units)` and it would not pick up a modified `astropy.constants` configuration.

Reply:
Reloading is quite flaky - I think one cannot easily change things that are already loaded inside a module.